src/index.html

```diff
@@ -12,6 +12,7 @@
       href="./styles/main.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file. Ensure that this SCSS file is compiled to a CSS file, and update the link to point to the resulting CSS file for proper styling.